### PR TITLE
Add BNB AA example & Add BNB testnet config

### PR DESCRIPTION
**Pull request by wsdt**
## Overview

Adding simple example integration test (will extend the example later) to test out BNB AA/Bundler on testnet. 

## Changes

- Add correct network configuration to AA package to be able to use on BNB testnet
- Add network name parameter for deterministic deployer to avoid using default config (= Goerli eth)
- Add integration test for BNB testnet + execute successfully. 
- Also includes a minor fix for the PR commenter

## Testing

Integration test runs successfully on BNB testnet.

### CODE REVIEW

File: boba_examples/aa-general-example/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Account Abstraction example
+
+Example for Boba BNB testnet to send userOperation to a wallet through the bundler (high level api).
```
Comment (InoMurko) [inline boba_examples/aa-general-example/README.md:3]: What kind of user operation?

**Review by InoMurko — APPROVED** [review]

File: boba_examples/aa-general-example/README.md (hunk 1 above)
Comment (InoMurko) [reply boba_examples/aa-general-example/README.md:3]: Question that needs an answer in these docs :-)

File: boba_examples/aa-general-example/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+This example conducts a simple ETH transfer.
```
Comment (souradeep-das) [inline boba_examples/aa-general-example/README.md]: should we say a simple contract call?

**Review by souradeep-das — APPROVED** [review]
looks nice! 
should we convert this from a test structure to a script structure with instructions for walkthroughs?
(also we could rename the pr from inegration test -> examples)

**Comment by wsdt** [comment]
> looks nice! should we convert this from a test structure to a script structure with instructions for walkthroughs? (also we could rename the pr from inegration test -> examples)

already on it! Should be done in no time, ideally we have both I would say :-)

**Comment by wsdt** [comment]
> > looks nice! should we convert this from a test structure to a script structure with instructions for walkthroughs? (also we could rename the pr from inegration test -> examples)
> 
> already on it! Should be done in no time, ideally we have both I would say :-)

will add both examples in separate PR. 

FYI

**Comment by souradeep-das** [comment]
sounds good, thanks! 🚀

**Pull request merged**